scripts/provision_lib_artefacts.sh

```diff
@@ -0,0 +1,27 @@
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Since we now use the `scripts` directory to contain any type of scripts (as opposed to only containing scripts that perform on-chain transactions, as was the case before), I would suggest to also move `deploy.sh` to this directory.

Reply:
Makes sense.